ghutil/ghutil.go

```diff
@@ -416,7 +420,7 @@ func (ghc *GitHubClient) ProcessPullRequest(ctx context.Context, orgName string,
 			}
 
 			if labelsUpdatedForNonCompliance {
-				addComment(pullRequestNonComplianceReason)
+				addReview(pullRequestNonComplianceReason, ReviewRequestChanges)
```

Comment:
Can we have the tool resolve its own review if the user updates their PR such that next time, it's a compliant PR?